pkg/minishift/util/setproxytoshell.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
> Let's follow Go file naming convention here set_proxy_to_shell.go

I don't think this is a Go convention, but the current code base uses underscores in file names, so I think we should stick to it. In actual code we of course use camel case.